janitor/functions/impute.py

```diff
@@ -1,13 +1,15 @@
 """Implementation of `impute` function"""
 from typing import Any, Hashable, Optional
 
+import lazy_loader as lazy
 import numpy as np
 import pandas_flavor as pf
 import pandas as pd
```

Comment:
I saw all packages in `janitor/math.py` were wrapped by `lazy.load`.
But `janitor/functions/impute.py` only wrapped `scipy.stats`.

Reply:
i didn't try to go beyond original @ericmjl code changes. 
it could be true that some newer additions on dev would require changes in imports. i assume we can address it later in subsequent PRs.

Reply:
Good point, @Zeroto521. We can continue to improve coverage of lazy loading, so no worries here. Plus we might also benefit from Python having lazy loading by default! (I'm thinking of [PEP690](https://peps.python.org/pep-0690/) here.) Let's stick with what we have for now.